packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -243,7 +244,16 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
     ev.stopPropagation();
   }
 
+  private _preventBounce(link: INavLink, ev: React.MouseEvent<HTMLElement>): void {
+    if (!link.url && link.forceAnchor) {
+      ev.preventDefault();
+    }
+  }
+
   private _onNavAnchorLinkClicked(link: INavLink, ev: React.MouseEvent<HTMLElement>): void {
+    // In this case, the href would be "#" and needs a preventDefault to prevent it from bouncing around
```

Comment:
Might be more accurate to say something like this:
```suggestion
    // If the href is "#" we should call preventDefault to prevent scrolling to the top of the page
```